.github/workflows/ci.yml

```diff
@@ -36,9 +36,12 @@ jobs:
           # TODO: Replace with go-version-from-file when it is supported
           # https://github.com/actions/setup-go/pull/62
           go-version: ${{ steps.go-version.outputs.content }}
+      - # since release builds specify the preload tag, we must run the generator
```

Comment:
I would say it's good to run this for each PR anyway as it's a way of testing the generator logic. 🤷🏻‍♂️

Reply:
Okay, it just adds a few minutes to the build but if it's useful then great!